standard_monoids.lua

```diff
@@ -1,17 +1,5 @@
 -- Standard effect monoids, to provide canonicity.
 
-local function mult(x, y) return x * y end
-
-local function mult_fold(elems)
-	local tot = 1
-
-	for k,v in pairs(elems) do
-		tot = tot * v
-	end
-
-	return tot
-end
-
```

Comment:
Instead of deleting mult_fold, could you make the definitions of the physics monoids use mult_fold instead?